setup.py

```diff
@@ -101,6 +102,9 @@
     include_package_data=True,
     platforms='any',
     entry_points={
+        'flask.commands': [
```

Comment:
Moderate: isn't it `console-scripts`? For example: https://github.com/inveniosoftware/invenio-cli/blob/master/setup.py#L77

Not sure if we want it under `invenio` or by itself

Reply:
`console-scripts` is for the name of the script by itself and `flask.commands` for the sub-commands under `invenio`. I think having it under `invenio` makes sense: we don't want to create multiple command-line tools (2 is enough :) ). With the current `setup.py` above, it works:

```
$ pipenv run invenio rdm-records fake
Creating demo records...
Created records!
$ curl -k -XGET https://localhost:5000/api/records/
{"aggregations":{"keywords":{"buckets":[],"doc_count_error_upper_bound":0,"sum_other_doc_count":0}},"hits":{"hits":[{"created":"2019-11-01T19:18:49.070162+00:00","id":"10","links":{"self":"https://localhost:5000/api/records/10"},"metadata":{"access_right":"open","contributors":[{"name":"Jason Collins"}],"description":"optimize leading-edge markets","owners":[1],"publication_date":"1978-03-30","recid":"10","resource_type":{"subtype":"dataset","type":"Dataset"},"title":"Rodriguez-Morgan's dataset"}...}
```